features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/label/label-add/js/edit.js

```diff
@@ -119,6 +119,21 @@ function validateInput(text, element, errorMsg) {
     }
 }
 
+function validateLength(text, element, errorMsg) {
+    var elementId = element.attr('id');
+    text = text.trim();
+    if(text.length > 255) {
```

Comment:
minor formatting error. add space after `if`. Fix in other places as well

Reply:
fixed in 7664836